### PR TITLE
Transform `export type {} from` to `export {} from`

### PR DESCRIPTION
**Pull request by lautis**
Handle cases where export declaration re-exports types from another file.

In Flow
```
export type { A, B } from "./dep.js";
```

becomes the following TypeScript:

```
export { A, B } from "./dep";
```

### CODE REVIEW

**Comment by k4b7** [comment]
This is important.  Thankfully going from Flow to TS is the easy direction for handling type imports.

File: src/transform.js [hunk 1]
```diff
@@ -6,6 +6,14 @@ const computeNewlines = require("./compute-newlines.js");
 const locToString = loc =>
   `${loc.start.line}:${loc.start.column}-${loc.end.line}:${loc.end.column}`;
 
+const stripSuffixFromImportSource = path => {
+  // TODO: make this configurable so we can output .ts[x]?
```
Comment (k4b7) [inline src/transform.js:10]: Can you do `import foo from "./foo.ts"`?  In the past when I've tried it complained so ended up always using `import foo from "./foo"` when writing TypeScript.

Comment (lautis) [reply src/transform.js:10]: I've successfully used `export foo from "./foo.ts"`, but that said, in this case I changed the flow files to "export *" to get rid of errors.

To double-check, are you suggesting that the export should be rewritten as following?

```
import { foo } from "./foo";
export { foo };
```

Or something else?

Comment (k4b7) [reply src/transform.js:10]: I wasn't suggesting any changes to this PR.  Just curious about the TODO.

Comment (k4b7) [reply src/transform.js:10]: ~Can create an issue for the TODO?~

Comment (k4b7) [reply src/transform.js:10]: I didn't realize that I wrote this TODO and you were just moving the code.  I can create a ticket for it.  😅

**Review by k4b7 — APPROVED** [review]
LGTM.  Thanks for the PR.

**Pull request merged**